src/ComposerRequireChecker/FileLocator/LocateComposerPackageSourceFiles.php

```diff
@@ -16,15 +16,22 @@
 use function ltrim;
 use function str_replace;
 
+/**
+ * @psalm-type ComposerConfig = array{vendor-dir?: string}
+ * @psalm-type ComposerRequire = array<string, string>
+ * @psalm-type ComposerAutoload = array{exclude-from-classmap?: list<string>, classmap?: list<string>, files?: list<string>, psr-0?: list<string>, psr-4?: list<string>}
```

Comment:
Let's use some multi-line declarations here: everything on a single line is not healthy :D

Reply:
Done.